README.md

```diff
@@ -16,8 +20,9 @@ A CakePHP plugin that provides a Shell to read an app's Configure vars from the
 
 ## Installation
 
-`composer require loadsys/config-read:~3.0`
-
+```bash
+$ composer require loadsys/config-read:~3.*
+```
```

Comment:
This is redundant now. The tilde operator seems to be preferable for our use. `~3.0` is equivalent to `>=3.0.0 <4.0`. See docs: https://getcomposer.org/doc/01-basic-usage.md#package-versions